docker-compose.yaml

```diff
@@ -55,7 +55,9 @@ services:
       - "30333:30333"
     volumes:
       - ./spec-data:/cereio/spec-data
-      - ./chain-data:/data
+      - type: volume
+        source: chainData
+        target: /chainData
```

Comment:
Why `target` start from `/` but `source` - no?

Reply:
The source points to the volume name, whereas the target points to the location where it should be created inside the container.